src/en/charms-storage.md

```diff
@@ -18,49 +23,50 @@ management abilities. Related commands are listed below, along with a brief
 description of each.
 
 [`add-storage`][commands-add-storage]
-: Adds unit storage dynamically.
+: Creates and attaches a storage instance to a unit.
 
 [`attach-storage`][commands-attach-storage]
-: Attaches existing storage to a unit.
+: Attaches an existing storage instance to a unit.
 
 [`create-storage-pool`][commands-create-storage-pool]
 : Creates or defines a storage pool.
 
 [`detach-storage`][commands-detach-storage]
-: Detaches storage from a unit.
+: Detaches a storage instance from a unit.
+
+[`remove-storage`][commands-remove-storage]
+: Removes a storage instance from a model.
```

Comment:
I think it would help disambiguation if this command and the one above explicitly said what happened to the actual storage (I know this is covered later, but this is higher up the page and may be the only things skim readers see)